inc/widgets/author-bio.php

```diff
@@ -25,28 +25,79 @@ function __construct() {
 	function widget( $args, $instance ) {
 
 		global $post;
-		$author_id = $post->post_author;
 
-		$output = '';
+		$authors = array();
+		$bios = '';
 
-		echo $args['before_widget'];
+		if ( is_singular() ) {
```

Comment:
The placement of `if ( is_singular() ) {}` is a little confusing, because if this is false, it basically shuts down the widget output - however that's not easily identifiable without reading the entire function.

Why don't we move that check to the top of the function just below `global $post`, formatted like:
```
// Return if not on a single post/page/attachment display
if ( ! is_singular() ) {
   return;
}
```